src/frontend/src/flows/login/index.ts

```diff
@@ -15,7 +38,38 @@ export const login = async (
   connection: AuthenticatedConnection;
 }> => {
   try {
-    const x = await tryLogin(connection);
+    const userNumber = getUserNumber();
+    const clearCache = () => {
+      const message = `This will forget your anchor number but your credentials will still be stored. To clear your credentials please check your device's documentation on how to clear WebAuthn credentials.`;
+      const confirmed = confirm(message);
+      if (confirmed) {
+        window.localStorage.clear();
+        window.location.reload();
+      }
```

Comment:
should we had a `return;` after `reload` just in case for an unknown reason reload would not work? most probably never gonna happen but just in case?

I guess not related to this refactoring